src/core/eme/session.ts

```diff
@@ -177,10 +156,15 @@ function sessionEventsHandler(
 
       return license.catch((error: Error) => {
         throw new EncryptedMediaError("KEY_STATUS_CHANGE_ERROR", error, true);
-      }) as Observable<LicenseObject>;
+      }).map((licenseObject) => {
+        return {
+          license: licenseObject as LicenseObject,
+          msg: "key-status-change" as "key-status-change",
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unnecessary cast. ([typescript:S4325](https://sonarcloud.io/organizations/rx-player/rules#rule_key=typescript%3AS4325))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rx-player&pullRequest=187&issues=AWLjaM-Pjr7wspRo8NBM&open=AWLjaM-Pjr7wspRo8NBM)

[ISSUE_KEY]: AWLjaM-Pjr7wspRo8NBM